common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -32,6 +32,11 @@
     private AuthenticationConstants() {
     }
 
+    /**
+     * Version name of ADAL.
+     */
+    public static final String ADAL_VERSION_NAME = "0.1";
```

Comment:
Why was this added?  This is the constants file for ADAL not for common so unclear on how this is intended to be used.